### PR TITLE
Upgrade to hawkular-build-tools 16

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@jpkrohling could you please review and eventually merge?

**Comment by ppalaga** [comment]
I have two followups ready that depend on the upcoming parent 39: 
- https://github.com/ppalaga/hawkular-commons/tree/p39
- https://github.com/ppalaga/hawkular-accounts/tree/p39.1

File: pom.xml [hunk 1]
```diff
@@ -234,7 +234,8 @@
     <version.com.datastax.cassandra>3.0.0</version.com.datastax.cassandra>
     <!-- keep in sync with modules/system/layers/base/com/fasterxml/jackson/core of the current WildFly/EAP -->
     <version.com.fasterxml.jackson.core>2.5.4</version.com.fasterxml.jackson.core>
-    <version.com.google.guava>16.0.1</version.com.google.guava>
+    <!-- keep in sync with modules/system/layers/base/com/google/guava of the current WildFly/EAP -->
+    <version.com.google.guava>18.0</version.com.google.guava>
```
Comment (jpkrohling) [inline pom.xml:238]: Does this work with the cassandra driver we use? I remember having conflicting versions, but not sure where it was.

**Comment by jpkrohling** [comment]
Other than the comment about the guava dependency, it looks good to me.

File: pom.xml (hunk 1 above)
Comment (ppalaga) [inline pom.xml:238]: > Does this work with the cassandra driver we use?

Yes, it does. The driver itself uses guava 18. The conflict we have is related to Titan used by inventory, which has to be fixed somehow (playing with guava versions is not an option anymore).

**Pull request merged**